src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink';
+import { SexFilter } from '../../types/SexFilter';
+
+type Props = {
+  query: string;
+  changeQuery: (input: string) => void;
+  sex: string | null;
+  centuries: string[];
+};
+
+const sexCollection: SexFilter[] = [
+  { id: 0, sex: null, title: 'All' },
+  { id: 1, sex: 'm', title: 'Male' },
+  { id: 2, sex: 'f', title: 'Female' },
+];
+
+const centuryCollection = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC<Props> = ({
+  query,
+  changeQuery,
+  sex,
+  centuries,
+}) => {
+  const changeCentury = (century: string) => (centuries.includes(century)
```

Comment:
The `changeCentury` function is better placed as a helper outside the component so that it is not re-created during each render.